test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -21,15 +21,25 @@
  * questions.
  */
 
-/* @test
- * @bug 8011536 8151430 8316304
+/* @test id=tmp
+ * @bug 8011536 8151430 8316304 8334339
  * @summary Basic test for creationTime attribute on platforms/file systems
- *     that support it.
+ *     that support it, test directory is /tmp.
  * @library  ../.. /test/lib
  * @build jdk.test.lib.Platform
  * @run main CreationTime
  */
 
+/* @test id=cwd
+ * @bug 8011536 8151430 8316304 8334339
+ * @summary Basic test for creationTime attribute on platforms/file systems
+ *     that support it, test directory is JTwork/scratch. The JTwork/scratch
+ *     directory maybe at diff disk partition to /tmp on linux
```

Comment:
I think you should want to say that it tests using the test scratch directory.

Reply:
The @summary description has been updated.